include/init.h

```diff
@@ -74,20 +74,20 @@ void z_sys_init_run_level(int32_t level);
  *
  * @param init_fn Address to the init function of the entry.
  *
- * @param device A device driver instance pointer or NULL
+ * @param dev_instance A device driver instance pointer or NULL
```

Comment:
why not dev or _device?

Reply:
Tried using dev and it is throwing an error as .dev is used in the macro. And to add a prefix, all the macro parameters need to be changed to maintain the consistency and was not sure if that's a good idea.

Reply:
I recommend renaming all the macro args to begin with a leading underscore, don't know if this is part of our coding style but it's the cleanest way to avoid collisions